.github/workflows/build.yml

```diff
@@ -82,6 +93,11 @@ jobs:
                 if: matrix.collections != ''
                 run: composer require "doctrine/collections:${{ matrix.collections }}" --no-update --no-scripts
 
+            -
+                name: Remove hateoas on Symfony 7
+                if: matrix.symfony == '^7.0'
+                run: composer remove --dev willdurand/hateoas-bundle --no-update --no-scripts
```

Comment:
Is this package still a problem? The newest version supports SF7 if I see correctly.
Ref.: https://packagist.org/packages/willdurand/hateoas-bundle#2.6.0

Reply:
Yes, there is an issue => https://github.com/willdurand/BazingaHateoasBundle/issues/108